docs/index.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ecosystem website</title>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/version/v2.0.1/tile.rtl.min.js"></script>
```

Comment:
```suggestion
    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/version/v2.0.1/tile.min.js"></script>
```

I missed this before, but you're actually importing the "right-to-left" versions of the tile (for languages such as arabic which read right-to-left). Removing this should fix the alignment issues with the expandable tile.